src/main/php/lang/ast/nodes/Property.class.php

```diff
@@ -3,6 +3,7 @@
 class Property extends Annotated implements Member {
   public $kind= 'property';
   public $name, $modifiers, $expression, $type, $holder;
+  public $hooks= null;
```

Comment:
We are not adding this as constructor parameter partly in order to prevent a BC break, partly because of how parsing works.